x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/TransportEsqlQueryAction.java

```diff
@@ -164,7 +164,7 @@ private void innerExecute(Task task, EsqlQueryRequest request, ActionListener<Es
             request.query(),
             request.profile(),
             request.tables(),
-            System.nanoTime()
+            System.nanoTime()  // MP TODO: remove this
```

Comment:
Should this be checked in?

Reply:
Thanks. I had another commit after this one to remove queryStartTimeNanos from Configuration but that causes bwc tests to fail so I had to revert it and this was left. I'll remove it now.